src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Formatting/FormattingOptions2.cs

```diff
@@ -39,20 +39,21 @@ public Provider()
 
         public static PerLanguageOption2<bool> UseTabs =
             new(FeatureName, FormattingOptionGroups.IndentationAndSpacing, nameof(UseTabs), LineFormattingOptions.Default.UseTabs,
-            storageLocation: new EditorConfigStorageLocation<bool>(
-                "indent_style",
-                s => s == "tab",
-                isSet => isSet ? "tab" : "space"));
+            storageLocations: ImmutableArray.Create<OptionStorageLocation2>(
+                new EditorConfigStorageLocation<bool>("indent_style", s => s == "tab", isSet => isSet ? "tab" : "space"),
+                new LocalClientSettingsStorageLocation("TextEditor.%LANGUAGE%.Insert Tabs")));
```

Comment:
So did this not work unless you did the request through the local variant? I would have thought these settings roam, do they not?

Reply:
Will follow up on whether these roam or not.